CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
+## 0.3.0
+
```

Comment:
Let's add a Breaking Changes note here about the JSONPath library since this will require users update their configs.

Reply:
Updated :+1: